admin-dev/themes/new-theme/js/pages/product/specific-price/form/index.ts

```diff
@@ -39,8 +38,6 @@ $(() => {
     'DateRange',
   ]);
 
-  const {eventEmitter} = window.prestashop.instance;
```

Comment:
The code here with CurrencySymbolUpdater should be cleaned as well
There are differences in this code from the generic component, so the component needs to provide a callback on symbol change, so that we can handle the specific switch use case for specific price here